challenge/src/main/java/com/ifood/challenge/location/LocationService.java

```diff
@@ -2,23 +2,23 @@
 
 import org.springframework.stereotype.Service;
 
-import java.util.List;
-import java.util.Random;
-
 @Service
 public class LocationService {
-    public List<Location> getLocation(int number){
+    public Location[] getLocation(int number){
         Location locations[] = new Location[number];
+        double doubleFirst = 1.000;
+        double doubleSecond = 2.000;
```

Comment:
The exercise states that random numbers should be generated:

https://rwondemand.atlassian.net/browse/DRVDXP-177

Reply:
I had forgotten that when I did the unit test, i change back